core/src/lib.rs

```diff
@@ -53,6 +53,9 @@ pub const ONE: Felt = Felt::ONE;
 /// The minimum length of the execution trace. This is the minimum required to support range checks.
 pub const MIN_TRACE_LEN: usize = 1024;
 
+/// Number of field elements in a Word.
+pub const WORD_LEN: usize = 4;
```

Comment:
nit: if we are introducing this here, might be good to update the definition of `Word` on line 40 above.